src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Class create to print items.
+ *
+ * @author Galvin Leow Wen Yuan
+ * @author A0200204J
+ * @version v1.0
+ */
+public class Ui {
+
+    /**
+     * Print out welcome message with the menu.
+     */
+    public void showWelcome() {
+        String
+                logo =
+                " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n" +
+                        "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo + "\nHello ! I'm Duke.\nWhat can I do for you?");
+        System.out.println(menu());
+    }
+
+    /**
+     * Return String that provide user with functionalities of the application.
+     *
+     * @return String menu
+     */
+    private String menu() {
```

Comment:
Perhaps getMenu would be a better method name as methods should be named using verbs.